tests.sh

```diff
@@ -1,3 +1,4 @@
+#/bin/bash -x
```

Comment:
Good point, but I wouldn't use `-x` here. This script already displays messages about its status.

Reply:
Pushed new version without -x